packages/rsbuild-plugin-angular/vitest.config.mts

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
     globals: true,
     environment: 'node',
     include: ['src/**/*.unit.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-
+    setupFiles: ['../../testing/vitest-setup/src/lib/fs-memfs-setup.ts'],
```

Comment:
This reduced mock boilerplate and gives more control over the file system in unit test.